code/modules/cm_marines/overwatch.dm

```diff
@@ -26,20 +26,25 @@
 	var/marine_filter = list() // individual marine hiding control - list of string references
 	var/marine_filter_enabled = TRUE
 	var/faction = FACTION_MARINE
+	var/obj/structure/orbital_cannon/current_orbital_cannon = null
```

Comment:
null on destroy

Reply:
not sure what is expected of me here

Reply:
to be fair it works the same way it did before, but I might be wrong